### PR TITLE
[13.0] [MIG] l10n_it_fiscalcode: Migration to 13.0

### PR DESCRIPTION
**Pull request by victoralmau**
Related to: https://github.com/OCA/l10n-italy/pull/1931 (14.0)

@Tecnativa TT27610

### CODE REVIEW

File: l10n_it_fiscalcode/README.rst [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fl10n--italy-lightgray.png?logo=github
+    :target: https://github.com/OCA/l10n-italy/tree/14.0/l10n_it_fiscalcode
+    :alt: OCA/l10n-italy
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/l10n-italy-14-0/l10n-italy-14-0-l10n_it_fiscalcode
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/122/14.0
```
Comment (OpenCode) [inline l10n_it_fiscalcode/README.rst:23]: Version 13.0, not 14.0

File: l10n_it_fiscalcode/README.rst [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+This module is part of the `OCA/l10n-italy <https://github.com/OCA/l10n-italy/tree/14.0/l10n_it_fiscalcode>`_ project on GitHub.
```
Comment (OpenCode) [inline l10n_it_fiscalcode/README.rst:115]: Version 13.0, not 14.0

File: l10n_it_fiscalcode/i18n/am.po [hunk 3]
```diff
@@ -0,0 +1,248 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# * l10n_it_fiscalcode
+#
+# Translators:
+# OCA Transbot <transbot@odoo-community.org>, 2018
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 11.0\n"
```
Comment (OpenCode) [inline l10n_it_fiscalcode/i18n/am.po:9]: Version 13.0, not 11.0

File: l10n_it_fiscalcode/i18n/ar.po [hunk 4]
```diff
@@ -0,0 +1,249 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# * l10n_it_fiscalcode
+#
+# Translators:
+# OCA Transbot <transbot@odoo-community.org>, 2018
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 11.0\n"
```
Comment (OpenCode) [inline l10n_it_fiscalcode/i18n/ar.po:9]: Version 13.0, not 11.0

**Review by OpenCode — CHANGES_REQUESTED** [review]

File: l10n_it_fiscalcode/README.rst (hunk 1 above)
Comment (victoralmau) [reply l10n_it_fiscalcode/README.rst:23]: Changed it now although but will regenerate it in merge process.

File: l10n_it_fiscalcode/README.rst (hunk 2 above)
Comment (victoralmau) [reply l10n_it_fiscalcode/README.rst]: Changed it now although but will regenerate it in merge process.

File: l10n_it_fiscalcode/i18n/am.po (hunk 3 above)
Comment (victoralmau) [reply l10n_it_fiscalcode/i18n/am.po:9]: It's not need to change it according to https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0

File: l10n_it_fiscalcode/i18n/ar.po (hunk 4 above)
Comment (victoralmau) [reply l10n_it_fiscalcode/i18n/ar.po:9]: It's not need to change it according to https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0

Comment (pedrobaeza) [reply l10n_it_fiscalcode/i18n/ar.po:9]: Yes, POT file will be regenerated after merge, and PO files will be auto-update if needed.

**Review by OpenCode — APPROVED** [review]

**Pull request closed without merging**